conference_instructions.md

```diff
@@ -0,0 +1,46 @@
+# Running Liberty Bikes at a Conference
+
+The following information can be used to setup the Liberty Bikes game on a standalone laptop and local wifi router for the purposes of demoing it at a conference. 
+
+
+# Hardware needed:
+
+1. Laptop with Liberty Bikes code
+2. Ethernet cable
+3. Wifi Router (ex, Linksys)
+4. Up to 4 mobile devices (one for each player)
+
+
+# Router setup (linksys)
```

Comment:
I don't think most people running this at a conf would be able to follow these steps -- it seems like pretty advanced stuff. When I did this at EclipseCon, we didn't have to mess around with network config at all really. We just connected everything up to the wifi network and used hardcoded IP addresses. Maybe instead we can explain how to check the `frontend` service's logs to determine what IP the endpoint is available at?

Reply:
I will play around next week with a setup similar to yours and see if I can simplify.